spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -27,6 +23,18 @@
              auth_headers_json: auth_headers.to_json,
              saved_claim_id: saved_claim.id)
     end
+    let(:open_claims_cassette) { 'evss/claims/claims_without_open_compensation_claims' }
+    let(:rated_disabilities_cassette) { 'evss/disability_compensation_form/rated_disabilities' }
+    let(:submit_form_cassette) { 'evss/disability_compensation_form/submit_form_v2' }
+    let(:cassettes) { [open_claims_cassette, rated_disabilities_cassette, submit_form_cassette] }
+
+    before do
+      cassettes.each { |cassette| VCR.insert_cassette(cassette) }
```

Comment:
Since every single spec in this file uses some combination of these VCR cassettes, I took the opportunity to refactor the cassettes themselves into configurable `let` variables, so that each spec can just provide any overrides for its own situation.

It does mean that the cassettes are decoupled from the code that uses them (a good thing in this case) so I switched from Ruby block style to before-insert/after-eject style.

Reply:
Yeah, I played around with capturing `allow_playback_repeats` as an additional variable, but since the tests are fine as is, I left that out.